projects/packages/jetpack-mu-wpcom/src/features/wpcom-themes/wpcom-themes.php

```diff
@@ -13,6 +13,9 @@
  * Displays a banner before the theme browser that links to the WP.com Theme Showcase.
  */
 function wpcom_themes_show_banner() {
+	if ( function_exists( 'is_agency_managed_site' ) && is_agency_managed_site() ) {
+		return;
+	}
```

Comment:
Hides the theme showcase banner.

<img src="https://private-user-images.githubusercontent.com/5634774/341847113-709a7159-3ab7-4779-9c9e-a8ce4d50bf4c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTkwMDkzOTcsIm5iZiI6MTcxOTAwOTA5NywicGF0aCI6Ii81NjM0Nzc0LzM0MTg0NzExMy03MDlhNzE1OS0zYWI3LTQ3NzktOWM5ZS1hOGNlNGQ1MGJmNGMucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDYyMSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA2MjFUMjIzMTM3WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MTg5NGRlOTdhZGQwYTVlZTExY2YzNGE4YzRhOTRlOGE4YzliYzhkYWEwODc1ZGUyZGIyOWIyOTFkNTg0NzJhMSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.VornLYwVI9UrfrS-Sp5BkysQ_At5I_Ttvu2SmEDjHNE" />